### PR TITLE
Add kludge for Uplink's scientist model from HL 25th anniversary update.

### PR DESCRIPTION
**Pull request by nekonomicon**

### CODE REVIEW

**Comment by a1batross** [comment]
This is such a hack. If best Valve programmers implemented it in engine, I don't think we really have a choice. :(

File: engine/server/sv_init.c [hunk 1]
```diff
@@ -693,6 +685,13 @@ deactivate server, free edicts, strings etc
 void SV_DeactivateServer( void )
 {
 	int	i;
+	const char	*cycle = Cvar_VariableString( "disconcfgfile" );
+
+	if( COM_CheckString( cycle ))
+		Cbuf_AddTextf( "exec %s\n", cycle );
+
+	if( COM_CheckString( sv.name ))
```
Comment (a1batross) [inline engine/server/sv_init.c:693]: sv.name is always valid, it doesn't need a check like this.

**Review by a1batross — CHANGES_REQUESTED** [review]

File: engine/server/sv_init.c (hunk 1 above)
Comment (a1batross) [reply engine/server/sv_init.c:693]: Other than this, it's fine to me.

Comment (nekonomicon) [reply engine/server/sv_init.c]: Nope.
It seems needed `COM_CheckStringEmpty` anyway.

Comment (nekonomicon) [reply engine/server/sv_init.c]: If I start hldemo1 map using `map hldemo1` cmd without this check I have such output in the console:
```
[19:38:04] >map hldemo1
[19:38:04] couldn't exec maps/_unload.cfg
[19:38:04] 1.33 Mb alloced by server packet entities
[19:38:04] Spawn Server: hldemo1
[19:38:04] loading maps/hldemo1.bsp
[19:38:04] Warning: FS_LoadImage: couldn't load "alpha_sky"
[19:38:04] Warning: FS_LoadImage: couldn't load "solid_sky"
[19:38:04] lighting: colored
[19:38:05] couldn't exec maps/_unload.cfg
[19:38:05] execing maps/hldemo1_load.cfg
[19:38:05] Warning: Unknown command "ui_cmd_on_connect"
[19:38:05]
GAME SKILL LEVEL:1
```

**Review by a1batross — APPROVED** [review]

**Pull request merged**